Source/SwiftLintFramework/Rules/ExplictSelfRule.swift

```diff
@@ -0,0 +1,286 @@
+//
+//  ExplictSelfRule.swift
+//  SwiftLint
+//
+//  Created by Ian Keen on 2016-10-06.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+func nonTriggering(type: String) -> [String] {
```

Comment:
all the implicitly `internal` members in this file that aren't needed in other files should be marked as `private` or `fileprivate`.